.claude/commands/gh-pr.md

```diff
@@ -0,0 +1,14 @@
+Create (or update) a Pull Request.
```

Comment:
this is just a baseline - we should extend this with our rules, but it does a pretty good job already